.github/workflows/release.yml

```diff
@@ -210,14 +214,24 @@ jobs:
           cp target/$target/production/space-acres $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS
 
           cp res/macos/space-acres.icns $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources
+          cp res/macos/run-space-acres.sh $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS
 
           cp res/macos/Info.plist $APP_PREFIX/$target/SpaceAcres.app/Contents/
           sed -i '' "s/%BUNDLE_VERSION%/$BUNDLE_VERSION/g" $APP_PREFIX/$target/SpaceAcres.app/Contents/Info.plist
           sed -i '' "s/%BUNDLE_BUILD%/$BUNDLE_BUILD/g" $APP_PREFIX/$target/SpaceAcres.app/Contents/Info.plist
 
           # 2. Copy and fix dependencies
           destDir=$APP_PREFIX/$target/SpaceAcres.app/Contents/Frameworks/lib
-          process_dependencies $target $destDir $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
+          process_dependencies $target $destDir $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres "../Frameworks" "true"
+
+          # 3. Copy svg dependencies
+          cp -r /opt/homebrew/lib/gdk-pixbuf-2.0/2.10.0/loaders $APP_PREFIX/$target/SpaceAcres.app/Contents/Frameworks/loaders
+          rm -rf $APP_PREFIX/$target/SpaceAcres.app/Contents/Frameworks/loaders/*.a
+          for loader in $APP_PREFIX/$target/SpaceAcres.app/Contents/Frameworks/loaders/*.so; do
+            process_dependencies $target $destDir $loader "../Frameworks" "false"
+          done
```

Comment:
Do you really actually need to do all of this? It would have been much more readable if you copied the files that are needed rather than copying everything and then deleting some stuff.

Like here:
https://github.com/subspace/space-acres/blob/3db9689e6646f3fd3b6b3d550fb43c42f7ccfa8b/.github/workflows/release.yml#L182-L183